modules/microshift-security-context-constraints-alert-eval.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * microshift_running_apps/microshift-authentication.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations
+
+The `PodSecurityViolation` alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by reviewing the Kubernetes API server audit logs. This procedure uses the `must-gather` tool to gather the audit logs and then searches for the `pod-security.kubernetes.io/audit-violations` annotation.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To gather the audit logs, enter the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather -- /usr/bin/gather_audit_logs
```

Comment:
I've opened a bug to track the missing logs, but for now let's tell users where they are and how to get them.  They can be accessed with another `oc adm` subcommand and parsed with the exact same filters as defined in this doc (i.e. line 30).

This PR should also include https://docs.openshift.com/container-platform/4.13/security/audit-log-view.html as additional information, excluding ofc the last section about using `must-gather`.  If so, it'd be important to note that the `--path` value in those docs won't work for microshift and should instead be `--path=kube-apiserver/`.

So the commands on the audit docs should read `oc adm node-logs --role=master --path=kube-apiserver/`. Example output:

```rhel-92.lab.local audit-2023-08-18T01-30-48.245.log
rhel-92.lab.local audit-2023-08-18T18-25-41.663.log
rhel-92.lab.local audit-2023-08-19T11-21-29.225.log
rhel-92.lab.local audit-2023-08-20T04-16-09.622.log
rhel-92.lab.local audit-2023-08-20T21-11-41.163.log
rhel-92.lab.local audit-2023-08-21T14-06-10.402.log
rhel-92.lab.local audit-2023-08-22T06-35-10.392.log
rhel-92.lab.local audit-2023-08-22T23-26-27.667.log
rhel-92.lab.local audit-2023-08-23T16-52-15.456.log
rhel-92.lab.local audit-2023-08-24T07-31-55.238.log
rhel-92.lab.local audit.log
```

And to parse them:
`oc adm node-logs --role=master --path=kube-apiserver/audit.log | jq 'some_filter'`.

It's probably also worth saying that for now, when submitting an sos report, users should include the audit.log as well.  It can be scp'd off the microshift host from `/var/log/kube-apiserver/audit*.log`. Since the file is owned by root, it would be a 2 step process:

From a remote host:
1. `ssh microshift@MICROSHIFT_HOST 'sudo cp /var/log/kube-apiserver/audit.log ~/ && chown microshift ~/audit.log'`
2. `scp microshift@MICROSHIFT_HOST:~/audit.log .`

I'll open a docs ticket to make sure we don't forget to update once the sos report includes these logs.